### ISSUE TITLE
Reducing K3s Org Equinix usage

### ISSUE THREAD

**Issue by jeefy** [k3s-io/k3s#9956]
Hello! Friendly neighborhood CNCF person here. 

K3s is currently using a significant amount of resources in Equinix. As they're bare metal instances, turning them off still leaves a reservation and burns unnecessary carbon so-to-speak.

- Can you work with @vielmetti and I on optimizing your usage? 
- Do you need all those nodes sitting idle? 
- Is there a way to spin up nodes as needed? 
- What's preventing you from using GitHub actions?

We tried reaching out via the original thread here: https://github.com/cncf/cluster/issues/148 and we've attempted to slack numerous folks, and gotten radio silence. :)

PS this is all related to https://contribute.cncf.io/resources/newsletters/2024-02-21-projectnewsletter/#equinix-updates and the resource usage policy linked further below. :) Thanks!

**Comment by brandond** [comment]
Thanks for opening an issue! I will reach out to our infra folks about releasing the previous generation of CI runners.

**Comment by cwayne18** [comment]
My apoloigies for missing this, we're getting right on it!

**Comment by vielmetti** [comment]
@cwayne18 Looking forward to hearing your progress on this.

**Comment by jeefy** [comment]
Heya! Quickly checking in here :) It looks like you're moving towards some sort of gitops-controlled workflow (yay!) 

Small reminder that machines in Equinix are bare metal and therefore even though they're powered off, they're still "Reserved" and go against our donation. When you're done and ready to move towards your new machines/setup, please delete the old ones. 

Thanks! If you have any other questions or are blocked on anything, please reach out to @vielmetti or myself

**Comment by brandond** [comment]
I've poked our infra team again to ask them to take the final steps to release these hosts. I think we were under the impression that this was going to occur on the 18th of last month, thanks for the reminder to check in on that.

**Comment by vielmetti** [comment]
Closing this out is fine; if something comes up in the future that needs further attention there will be a new issue + a link back to this one.
